package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wazo-ts-types",
+  "version": "1.0.0",
+  "description": "TS types for Wazo APIs",
+  "main": "index.js",
+  "author": "Wazo (http://wazo.io)",
+  "license": "GPL-3.0-only",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "engines": {
+    "node": "22.x"
+  },
+  "scripts": {
+    "build": "npm run generate && npm run compile",
+    "compile": "rm -rf ./dist && tsc --project tsconfig.json",
+    "fix:confd": "sed -i -e 's#\"\"Firstname Lastname\" < number >#\"\\\\\\\"Firstname Lastname\\\\\\\" < number >#g' ./src/schemas/confd.ts",
+    "generate": "rm -rf ./src/schemas && node --experimental-strip-types ./src/build.ts && npm run fix:confd",
```

Comment:
Can we extract this command into a bash file? If we have many post compile scripts to execute if will be cleaner and centralized.

We could even call it at then end of `build.ts` so package.json stay clean